apps/assets/server/models/cloudinary.js

```diff
@@ -0,0 +1,47 @@
+const { getSecret } = require("../../helpers/secrets-management");
+const cloudinary = require('cloudinary').v2;
+
+class Cloudinary {
+  constructor(storage) {
+    this.name = storage.name;
+    this.ready = getSecret(storage.tenant, storage.authentication).then((decrypted) => {
+      const auth = decrypted.value;
+      cloudinary.config({
+        cloud_name: storage.metadata.bucketName,
+        api_key: auth.accessKeyId,
+        api_secret: auth.secretAccessKey,
```

Comment:
use Cloudinary's names for the variables from the secrets. e.g.: apiKey, apiSecret.
prefer even to use the same name from their UI interface at Cloudinary's dashboard.

Reply:
done